.dockerignore

```diff
@@ -0,0 +1 @@
+docker/
```

Comment:
Maybe add `.dockerignore` and `docker/` to `.gitignore` ?

Reply:
docker/, yes, .dockerignore, no, I think.

It's a different ignore mechanism. Because we are doing `docker build .`, everything in the repo is considered part of the docker build context. If the cache dir is not in .dockerignore we'd risk docker deciding to rebuild (because its build context is different) instead of using the cache in it.

Reply:
You are right, just `docker/` is enough.

Reply:
(Sorry the comment was pending...)